core/store/src/migrations.rs

```diff
@@ -320,3 +326,126 @@ pub fn migrate_13_to_14(path: &String) {
 
     set_store_version(&store, 14);
 }
+
+pub fn migrate_14_to_15(path: &String) {
+    let store = create_store(path);
+    let trie_store = Box::new(TrieCachingStorage::new(store.clone(), TrieCache::new(), 0));
+    let trie = Trie::new(trie_store, 0);
+
+    let mut store_update = store.store_update();
+    let batch_size_limit = 10_000_000;
+    let mut batch_size = 0;
+
+    for (key, value) in store.iter_without_rc_logic(DBCol::ColOutcomesByBlockHash) {
+        let block_hash = CryptoHash::try_from_slice(&key).unwrap();
+        let block =
+            store.get_ser::<Block>(DBCol::ColBlock, &key).unwrap().expect("block should exist");
+
+        if block.chunks()[0].height_included() == block.header().height() {
```

Comment:
It seems that you only migrate the first shard. Sharded networks (guildnet) will get into a broken state after this migration, right?